.cirrus.yml

```diff
@@ -29,12 +29,6 @@ env:
     # test changes made there, in a PR in this repository.
     SKOPEO_PR:
 
-    ####
-    #### Cache-image names to test with (double-quotes around names are critical)
```

Comment:
Should the double-quote note be preserved because it still applies to the image names below?